arch/arm64/configs/sun50iw1p1smp_linux_defconfig

```diff
@@ -3319,80 +3319,28 @@ CONFIG_DEFAULT_MESSAGE_LOGLEVEL=4
 CONFIG_FRAME_WARN=2048
 CONFIG_MAGIC_SYSRQ=y
 # CONFIG_STRIP_ASM_SYMS is not set
-# CONFIG_READABLE_ASM is not set
 CONFIG_UNUSED_SYMBOLS=y
 CONFIG_DEBUG_FS=y
 # CONFIG_HEADERS_CHECK is not set
 # CONFIG_DEBUG_SECTION_MISMATCH is not set
-CONFIG_DEBUG_KERNEL=y
-# CONFIG_DEBUG_SHIRQ is not set
-CONFIG_LOCKUP_DETECTOR=y
```

Comment:
DEBUG kernel is for DEBUG builds... To speed up things a RELEASE build is better ==> CONFIG_DEBUG_KERNEL was removed. (Requires NOT to have CONFIG_EMBEDDED and NOT to have CONFIG_EXPERT)

Other parameter depend on this and are not used if no DEBUG build is done